docs/cookbook.rst

```diff
@@ -129,6 +129,31 @@ All tasks in CumulusCI are python classes that subclass `cumulusci.core.tasks.Ba
 
 For most tasks, you'll want to override the `_run_task` method in your subclass to provide the implementation. The return value of this function is saved as part of the StepResult. Exceptions from `cumulusci.core.exceptions` should be raised to communicate task status to the user or flow. If no exceptions are thrown, the task is considered to have completed successfully.
 
+Pydantic Task options
```

Comment:
@prescod - Let me know how this looks. I could probably come up with a better example task if you pressed me, just wanted to include only what is needed from a `cci` perspective and point users to Pydantic docs where applicable.